src/schema.graphql

```diff
@@ -1,23 +1,13 @@
 type Query {
   isLoggedIn: Boolean!
+  currentUser: User
   statistics: Statistics!
 }
 type Mutation {
-  login(email: String!, password: String!): LoggedInUser
+  login(email: String!, password: String!): String!
```

Comment:
This is not okay, I want the user back when logging in!!!

Reply:
@appinteractive as I said: The former `LoggedInUser` was only a **subset** of the `User` plus the token. This caused headache on the frontend because we had two different truths where e.g. the `name` might come from: Do we have keep track of the currently authenticaed `loggedInUser` or  `User` object?

The `loggedInUser` with the previous implementation, by design, could only be as subset what was encoded in the JWT token. I implemented the query `currentUser` to re-use the generated cypher query of `neo4j-graphql-js` filtered for the id from the JWT payload, thus returning a whole `User` object. Removing `loggedInUser` for `User`, it is unambiguous where `name` comes from. Plus, we can query for all attributes including `commentCount` etc. on the client. If we would encode the whole user object in the JWT payload that would be quite a lot for sth. that we want to pass around on every request.

To make this difference absolutely clear, I changed `login` to return the JWT only. This caused errors before and I don't want to put the original reason for this error back in again.

Reply:
Why not returning login type o object with the token and the whole user object?

`
{
  token: String!,
  User: User!
}
`

Reply:
That's right. I thought about that option. We could even make use of graphql fragments here. However I found it cumbersome to unpack the `token` from the returned object in the frontend. You can see [in the corresponding PR in the frontend](https://github.com/Human-Connection/Nitro-Web/pull/199) that we always unpack the token from the response. Considering that we have just one extra request after the login, I weighed complexity over performance.
Is that OK @appinteractive ?

Reply:
Lets talk about that. I think we have to actually speek tomorrow to get the point.